lib/Raven/Configuration.php

```diff
@@ -695,27 +659,29 @@ public function getErrorTypes()
      */
     public function setErrorTypes($errorTypes)
     {
-        $options = array_merge($this->options, ['error_types' => $errorTypes]);
-
-        $this->options = $this->resolver->resolve($options);
+        $this->mergeAndResolve(['error_types' => $errorTypes]);
     }
 
     /**
      * Configures the options for this processor.
      *
      * @param OptionsResolver $resolver The resolver for the options
+     *
+     * @throws \Symfony\Component\OptionsResolver\Exception\UndefinedOptionsException
+     * @throws \Symfony\Component\OptionsResolver\Exception\AccessException
      */
     private function configureOptions(OptionsResolver $resolver)
     {
         $resolver->setDefaults([
             'send_attempts' => 6,
             'trust_x_forwarded_proto' => false,
             'prefixes' => explode(PATH_SEPARATOR, get_include_path()),
+            'serializer' => new Serializer(),
+            'reprSerializer' => new Serializer(),
```

Comment:
As all options are named in lowercase with underscores this should be changed to follow that convention

Reply:
How can we use the dependency injection? There are at least two different classes that needs them, so I need to pass the same object twice.

Reply:
What I meant is that imho it would be better to add some methods to set the serializer to the client builder instead of treating them as options of the configuration and then be passed to the client as constructor arguments. I don't see the serializer to fit well into an array of options that is the configuration. Plus this way if an user doesn't want to use the client builder but instead prefers to create the client instance his own way it's free to do it